README.md

```diff
@@ -1,25 +1,437 @@
+```
+
+### Custom Event Handler
+
+The addEventListener method listens to events across all FusionCharts instances on a page and executes custom functions when an event is triggered.
+
+The generic method implemented above can be used to add a custom event listener which invokes a callback method and here it is: 
+```
+await fusionChartsService.CallFusionChartsFunction("addEventListener", "CHART_ID", "callback", "dataPlotClick", "function() {console.log('Custom method for event handling using generic method')}");
+```
+
+Below code snippet demonstrates the callback method implementation upon the event trigger.
+```
+if (functionName === 'addEventListener' || args[0][0] === "callback") {
+    let event = args[0][1]
+    let functionAsString = args[0][2];
+    let callbackFn = parseFunction(null, functionAsString);
+    var result = currentChart[functionName].call(currentChart, event, callbackFn);
+    return "Undefined";
+  }
+
+```
+
+
+## Fixes
+
+### Time Series Method
+setDataStore() : This mehtod is specific to Time Series Charts which takes data and schema from as input from the external files and creates a data source and a data table so as to render the chart.
+
+Initially an empty data source is created in the Index.razor file as shown:
+```
+myDataSource.data = new {};
+```
+
+Below is the code snippet to read the contents of the external data and schema files. Also the method setDataStore() is invoked:
+
+```
+String dataFilePath = "C:/Users/Harsh Jain/Desktop/Blazor FC Integration/blazor-fusioncharts/BlazorApp1/wwwroot/data.json";
+String schemeFilePath = "C:/Users/Harsh Jain/Desktop/Blazor FC Integration/blazor-fusioncharts/BlazorApp1/wwwroot/schema.json";
+String dataContent = File.ReadAllText(dataFilePath);
+String schemaContent = File.ReadAllText(schemeFilePath);
+
+await fusionChartsService.setDataStore("chartId", dataContent, schemaContent1);
+```
+
+Upon invoking the setDataStore() method above, it leads to the below code snippet in the FusionChartsService.cs file which in turn invokes another user-defined method in the blazor-fusioncharts.js:
+```
+public async Task setDataStore(String id, params object[] args){
+            await _jsruntime.InvokeVoidAsync("FusionCharts.setDataStore", id, args);
+        }
+```
+
+Finally, the functionality of setDataStore is implemented in the blazor-fusioncharts.js.\
+The code includes the creation of new data store and the parsing of data and schema to json.\
+A new data table is created with the data and schema and rendered it to the chart.
+
+```
+window.FusionCharts.setDataStore = (id, args) =>  {
+
+    let currentChart = FusionCharts(id);
+    let fusionDataStore = new FusionCharts.DataStore();
+    
+    let data = JSON.parse(args[0]);
+    let schema = JSON.parse(args[1]);
+
+    let fusionTable = fusionDataStore.createDataTable(data, schema);
+
+    currentChart.setChartData({data: fusionTable});
+};
+```
+
+### Resize Method
```

Comment:
Leave a note to the user that the user themselves can create more functions to simplify their use case.

Reply:
Done